_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+### Implementing the Policy on Service and Digital
+
+Crucially, this strategy helps the ESDC achieve the high-level objectives of the Policy on Service and Digital:
+
+- Client service experience and government operations are improved through digital transformation approaches.
+- Guided by a commitment to the guiding principles and best practices of the GC Digital Standards: design with users; iterate and improve frequently; work in the open by default; use open standards and solutions; address security and privacy risks; build in accessible from the start; empower staff to deliver better services; be good data stewards; design ethical services; collaborate widely.
+
+The benefits of using open standards and OSS (the code or specification is public, free to reuse and collaboratively adapt and improve; it can be inspected for security issues, is independent from vendors, and gravitates towards interoperable systems) clearly underpin these objectives.
+
+The Policy on Service and Digital makes it clear that working in the open and collaborative working methods will be the norm within the GC's IT community to foster the sharing of code, data and solutions.
+The principal working methods encouraged by this strategy are open, inclusive and co-creative.
+
+To drive efficiencies across the GC, the Policy on Service and Digital focuses on the client, ensuring proactive consideration at the design stage of key requirements of these functions in the development of operations and services.
+It establishes an enterprise-wide, integrated approach to governance, planning and management.
+Overall, the Policy on Service and Digital advances the delivery of services and the effectiveness of government operations through the strategic management of government information and data and leveraging of information technology, supporting the mandate of the Minister for Digital Government in leading the GC's digital transition.
+
+The Directive on Service and Digital refers to cyber security of digital solutions and services.
+OSS code can be inspected in detail and is easily included in systematic vulnerability scanning.
+This is another practical link with the Policy and Directive on Service and Digital.
+
+Notes:
+
+Canada has established clear vision and designed it policies to nudge departments towards it.
+By adopting this strategy, ESDC is taking another step in realizing the vision outlined by the government.
+
+### Sharing and reuse benefits all
+
+Open standards and OSS make it easy to share and reuse solutions, as well as data, information and knowledge.
+This is also in line with the Policy on Service and Digital and the Mandatory Procedures for Enterprise Architecture (EA) Assessment:
+
+- Data should be shared openly by default as per the Directive on Open Government
+- All source code must be released under an appropriate OSS license
+- Leverage and reuse existing solutions, components, and processes
+
```

Comment:
Maybe bold a bit, but we can state that this could result in promoting new commercial opportunities for Canadians (eg. support contracts) to the Gov and other business internationally.